contracts/Safe.sol

```diff
@@ -40,11 +41,12 @@ contract Safe is
     SecuredTokenTransfer,
     ISignatureValidatorConstants,
     FallbackManager,
-    StorageAccessible
+    StorageAccessible,
+    ISafe
 {
     using SafeMath for uint256;
 
-    string public constant VERSION = "1.4.1";
+    string public constant override VERSION = "1.4.1";
```

Comment:
Unrelated to this PR, but this should probably be 1.5.0 right?

Reply:
No changes required then 👍🏾